MapboxDirections/MBRoute.swift

```diff
@@ -194,6 +197,11 @@ open class Route: NSObject, NSSecureCoding {
      Each route produced by a single call to `Directions.calculate(_:completionHandler:)` has the same route identifier.
      */
     @objc open var routeIdentifier: String?
+    
+    /**
+     The locale to use for spoken instructions.
+     */
+    @objc open var spokenLocale: Locale?
```

Comment:
Rename this property to `speechLocale` or `voiceLocale` to better emphasize that this locale gets plugged into MapboxSpeech or the Voice API.